### PR TITLE
9 delete api comments comment

### PR DESCRIPTION
**Pull request by smlbrd**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -337,6 +337,28 @@ describe('POST /api/articles/:article_id/comments', () => {
   });
 });
 
+describe('DELETE /api/comments/comment_id', () => {
+  test('204: Responds with status code after deleting given comment_id from table', () => {
+    return request(app).delete('/api/comments/2').expect(204);
+  });
```
Comment (nc-christian-dutton) [inline __tests__/app.test.js:343]: Well done, on a delete endpoint we can't send anything on body so just the 204 is enough :)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -337,6 +337,28 @@ describe('POST /api/articles/:article_id/comments', () => {
...
+  test("404: Responds with an error message if comment_id doesn't exist", () => {
+    return request(app)
+      .delete('/api/comments/99999')
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Not Found');
+      });
+  });
+});
```
Comment (nc-christian-dutton) [inline __tests__/app.test.js:360]: Good error testing, all covered here!

File: db/seeds/utils.js [hunk 3]
```diff
@@ -22,3 +23,22 @@ exports.formatComments = (comments, idLookup) => {
...
+
+  return db.query(queryString, [value]).then(({ rows }) => {
+    if (rows.length === 0) {
+      return Promise.reject({
+        status: 404,
+        msg: `Not Found`,
+      });
+    }
+  });
+};
```
Comment (nc-christian-dutton) [inline db/seeds/utils.js:44]: This is a really smart util function, great work!

**Comment by nc-christian-dutton** [comment]
Have you updated your endpoints.json? 👀

**Comment by smlbrd** [comment]
> Have you updated your endpoints.json? 👀

I have now!! :sweat_smile:

**Pull request merged**